server/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java

```diff
@@ -200,7 +201,6 @@ public boolean isPrivateSetting(String key) {
         switch (key) {
             case IndexMetaData.SETTING_CREATION_DATE:
             case IndexMetaData.SETTING_INDEX_UUID:
-            case IndexMetaData.SETTING_VERSION_CREATED:
```

Comment:
you just converted one setting here to reduce the footprint right?

Reply:
Yup!

>We do not cutover all settings that we had been treating as private in this pull request, it is already quite large due to moving some tests around to account for the fact that some tests need to be able to set the index.version.created. This can be done in a follow-up change.